taskcat/_cli_modules/delete.py

```diff
@@ -0,0 +1,53 @@
+# pylint: disable=duplicate-code
+import logging
+
+from taskcat._cfn.stack import Stack
+from taskcat._cfn.threaded import Stacker
+from taskcat._client_factory import Boto3Cache
+
+LOG = logging.getLogger(__name__)
+
+
+class Delete:
+    """[ALPHA] Deletes an installed package in an AWS account/region"""
```

Comment:
I have concerns around the term "package".

Reply:
agree, struggling to come up with decent naming conventions, let's discuss

Reply:
We've spoken and resolved this contention point